yarn-project/sequencer-client/src/publisher/l1-publisher.ts

```diff
@@ -270,14 +301,55 @@ export class L1Publisher {
     return false;
   }
 
+  async commitTimeJump(slot: bigint) {
```

Comment:
i guess it needs to go in here to make the other tests pass within IS_DEV_NET

Reply:
The reason it was in the publisher itself was that it was something we needed to do when a block was published (or for the simulation issue), so keeping it there gave better control. 

Alternatively could be running in the background in the tests to progress blocks if nothing have happened for 12 seconds essentially making a fallback to the interval mining.

Reply:
Tried it out. Works fine for handling the "slot already filled", but non-obvious how to check for the simulation failure without rebuilding a lot of the sequencer, so just had a "progress block at least every 12 seconds". It will kinda work, but it is really slow if it start running into the "have to wait" case.